lib/commands/command-export-shell-version.bash

```diff
@@ -2,19 +2,50 @@
 # shellcheck source=lib/functions/versions.bash
 . "$(dirname "$(dirname "$0")")/lib/functions/versions.bash"
 
+unset_all_fish(){
+  printf "for var in (env | awk -F= '/^ASDF_[A-Za-z0-9_]+_VERSION=/ {print \$1}')\n"
+  printf "    echo Removing Env: \$var\n"
+  printf "    set -e \$var\n"
+  printf "end\n"
+}
+unset_all(){
+ printf "for var in \$(env | awk -F= '/^ASDF_[A-Za-z0-9_]+_VERSION=/ {print \$1}'); do\n"
+ printf "    echo Removing Env: " 
+ printf "    \$var\n"
+ printf "    unset \$var\n"
```

Comment:
```suggestion
 printf "    unset -v \$var\n"
```

The `-v` is needed to make sure Bash, Zsh, etc. implementations match closer in behavior, across versions.